src/showcase/showcase-directive.js

```diff
@@ -10,7 +10,7 @@
                     selection: '&'
                 },
                 link: function(scope, element) {
-                    var chart = sc.app(),
+                    var chart = sc.app().quandlApiKey('kM9Z9aEULVDD7svZ4A8B'),
```

Comment:
I copied the key from the Quandl service just to keep the changeset small - I expect you may want to update this to get the key from the service (or some other way, which doesn't involve duplicating the string)